contracts/marketplace/src/msg.rs

```diff
@@ -159,6 +159,12 @@ pub enum QueryMsg {
         current_time: Timestamp,
         token_id: TokenId,
     },
+    /// Get renewal price for multiple names
+    #[returns(Vec<(TokenId, Coin, Option<Bid>)>)]
```

Comment:
It's better to turn this tuple into its own struct since there's a bunch of data in here.

Reply:
yeah, def less of a pain to deal with on the TS/FE side for sure

Reply:
converted to return vector of structs in last commit